File: Python/quatro.py
numero = [[None, None], [None, None]]

numero[0][0] = int(input("Insira um valor: "))

for l in range(0, 2):
  for c in range(0, 2):
    numero[l][c] = numero[0][0]
    print(f"{numero[l][c]}")

File: JS/quatro.js
const numero = [];

const input = Number(prompt("Insira um número: "));
numero.push([ input ])


for(let linha=0; linha<2; linha++){
  for(let coluna=0; coluna<2; coluna++){    
    if(numero[linha]) {
        numero[linha].push(numero[0][0])
      } else {
        numero.push([ numero[0][0] ])
      } 
  }
}

  for(let l = 0; l < 2; l++){
    for(let c = 0; c < 2; c++){ 
      console.log(numero[l][c]);
    }
  }